packages/schema/lib/schemas/InputFieldSchema.js

```diff
@@ -30,8 +32,12 @@ module.exports = makeSchema(
       },
       dynamic: {
         description:
-          'A reference to a trigger that will power a dynamic dropdown.',
-        $ref: RefResourceSchema.id,
+          'A reference to a trigger, request, or function that will power a dynamic dropdown.',
+        oneOf: [
+          { $ref: RefResourceSchema.id },
+          { $ref: RequestSchema.id },
+          { $ref: FunctionSchema.id },
+        ],
```

Comment:
Initially wanted to suggest we add usage examples for the 3 options, but it seems a more detailed examples in https://docs.zapier.com/platform/home would do a better job.